config.fullstack.test.yaml

```diff
@@ -0,0 +1,251 @@
+                  recommendation:
+                    host: https://recommendation-api-beta.wmflabs.org
+                  purged_cache_control: test_purged_cache_control
+                  # Cache control for purged endpoints allowing short-term client caching
+                  purged_cache_control_client_cache: test_purged_cache_control_with_client_caching
+                  pdf:
+                    # Cache PDF for 5 minutes since it's not purged
+                    cache_control: s-maxage=600, max-age=600
+                    uri: https://proton-beta.wmflabs.org
+                  transform:
+                    cx_host: https://cxserver-beta.wmflabs.org
+                  skip_updates: false
+              - path: projects/proxy.yaml
+                options:
+                  backend_host_template: '{{"/{domain}/sys/legacy"}}'
+                  block_external_reqs: true
+          /{api:sys}: &default_sys
+            x-modules:
+              - path: projects/sys/default.wmf.yaml
+                options: *default_options
+              - path: projects/proxy.yaml
+                options:
+                  backend_host_template: '{{"/{domain}/sys/legacy"}}'
+                  block_external_reqs: false
+              - spec:
+                  paths:
+                    /legacy/key_value:
+                      x-modules:
+                        - path: sys/key_value.js
+                    /legacy/page_revisions:
+                      x-modules:
+                        - path: sys/page_revisions.js
+                    /table: &table_spec
+                      x-modules:
+                        - path: sys/table.js
+                          options:
+                            conf:
+                              backend: '{env(RB_TEST_BACKEND, sqlite)}'
+                              hosts: [localhost]
+                              keyspace: system
+                              username: cassandra
+                              password: cassandra
+                              defaultConsistency: one # or 'localQuorum' for production
+                              storage_groups:
+                                - name: test.group.local
+                                  domains: /./
+                              dbname: test.db.sqlite3 # ignored in cassandra, but useful in SQLite testing
+
+wikimedia_project: &wikimedia_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - path: projects/v1/wikimedia.wmf.yaml
+                options:
+                  <<: *default_options
+                  pageviews:
+                    host: https://wikimedia.org/api/rest_v1/metrics
+              - path: projects/proxy.yaml
+                options:
+                  backend_host_template: '{{"/{domain}/sys/legacy"}}'
+                  block_external_reqs: true
+          /{api:sys}:
+            x-modules:
+              - path: projects/proxy.yaml
```

Comment:
Why the double proy both here and in v1? We just need the one in v1 and then mount key_val and page_revs normally under sys/

Reply:
This is one thing I wanna discuss.. It's because we've moved `JSON.parse(res.body.toString('utf8'))` into the proxy - so now modules don't support getting the Buffer from the backend anymore. Not sure what to do here.